pipelines/curate_identity.py

```
@@ -560,3 +985,353 @@ def curate_records(records: Iterable[SourceRecord]) -> tuple[CurationResult, ...
+    for path in files:
+        rel = path.relative_to(root).as_posix()
+        text = path.read_text(encoding="utf-8")
+        for line_no, line in enumerate(text.splitlines(), 1):
+            if not line.strip():
+                continue
+            try:
+                obj = json.loads(line)
+            except json.JSONDecodeError as exc:
+                raise IdentityCurationError(
+                    f"JSON parse error at {rel}:{line_no}: {exc}"
+                ) from exc
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)